packages/suite/src/views/dashboard/components/AssetsCard/index.tsx

```diff
@@ -89,62 +128,77 @@ const AssetsCard = () => {
+                        color={isTableMode ? colors.BG_GREEN : colors.TYPE_LIGHT_GREY}
+                    />
+                    <Icon
+                        icon="GRID"
+                        onClick={() => setIsTableMode(false)}
+                        color={!isTableMode ? colors.BG_GREEN : colors.TYPE_LIGHT_GREY}
+                    />
+                </ActionsWrapper>
+            }
         >
-            <StyledCard>
-                <AnimatePresence initial={false}>
-                    <Grid>
-                        <Header>
-                            <Translation id="TR_ASSETS" />
-                        </Header>
-                        <Header>
-                            <Translation id="TR_VALUES" />
-                        </Header>
-                        <Header>
-                            <Translation id="TR_EXCHANGE_RATE" />
-                        </Header>
-                        {networks.map((symbol, i) => {
-                            const network = NETWORKS.find(
-                                n => n.symbol === symbol && !n.accountType,
-                            );
-                            if (!network) {
-                                return 'unknown network';
-                            }
-
-                            const assetBalance = assets[symbol].reduce(
-                                (prev, a) => prev.plus(a.formattedBalance),
-                                new BigNumber(0),
-                            );
-
-                            const assetFailed = accounts.find(
-                                f => f.symbol === network.symbol && f.failed,
-                            );
-
-                            return (
-                                <Asset
-                                    key={symbol}
-                                    network={network}
-                                    failed={!!assetFailed}
-                                    cryptoValue={assetBalance.toFixed()}
-                                    isLastRow={i === networks.length - 1}
-                                />
-                            );
-                        })}
-                        {discoveryInProgress && <AssetSkeleton />}
-                    </Grid>
-                </AnimatePresence>
-
-                {isError && (
-                    <InfoMessage>
-                        <Icon
-                            style={{ paddingRight: '4px', paddingBottom: '2px' }}
-                            icon="WARNING"
-                            color={theme.TYPE_RED}
-                            size={14}
-                        />
-                        <Translation id="TR_DASHBOARD_ASSETS_ERROR" />
-                    </InfoMessage>
-                )}
-            </StyledCard>
+            {!isTableMode && (
+                <GridWrapper>
+                    {assetsData.map(asset => {
+                        return (
```

Comment:
The explicit return statement can be omitted here.
```
{assetsData.map(asset => (
    <AssetGrid
        key={asset.symbol}
        network={asset.network}
        failed={asset.assetFailed}
        cryptoValue={asset.assetBalance.toFixed()}
    />
))}
```

Reply:
done